doc/tutorial/alignment_file/alignment_file_read_cigar.cpp

```diff
@@ -30,15 +30,13 @@ write_file_dummy_struct go{};
 #include <seqan3/io/alignment_file/all.hpp>
 #include <seqan3/std/filesystem>
 
-using namespace seqan3;
-
 int main()
 {
     std::filesystem::path tmp_dir = std::filesystem::temp_directory_path(); // get the temp directory
 
-    alignment_file_input fin{tmp_dir/"my.sam", fields<field::CIGAR>{}};
+    seqan3::alignment_file_input fin{tmp_dir/"my.sam", seqan3::fields<seqan3::field::CIGAR>{}};
 
     for (auto & [cigar] : fin)
```

Comment:
Can you rename the variable `cigar` to `cigar_vector? Do this in a separate commit. It will fix #1344 :) You can put `fixes  #1344` int he commit message, then it is linked.